Setup_EnvironmentGUI.py

```diff
@@ -0,0 +1,97 @@
+import pygame
+import sys
+
+import Control_EnvironmentGUI as Control_Environment
+import Setup_settingsGUI
+
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+WINDOW_HEIGHT = 770
+WINDOW_WIDTH = 1500
+SCREEN = None
+CLOCK = None
+
+
+class SetupEnvironment:
+    """
+    This will set up the actual environment for the simulation to be displayed in
+    """
+    def __init__(self):
+        global SCREEN, CLOCK
+        pygame.init()
+        SCREEN = pygame.display.set_mode((WINDOW_WIDTH, WINDOW_HEIGHT))
+        CLOCK = pygame.time.Clock()
+        SCREEN.fill(WHITE)
```

Comment:
I think it would be better looking if the grid has a black background with very thin lighter gridlines. For now the gridlines give us a good indication of where things are, but it may be a good idea in the future to make a grid toggle. When there's a lot of organisms on screen it's going to look nicer without the gridlines probably.